src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -599,8 +595,7 @@
     <Field Name="Checked" Type="bool"/>
     <Field Name="ExplicitCastInCode" Type="bool"/>
     <Field Name="ConstantValueOpt" Type="ConstantValue" Null="allow"/>
-    <!-- True for explicit casts with top-level `?`. Used by NullableWalker. -->
-    <Field Name="IsExplicitlyNullable" Type="bool" Null="allow"/>
+    <Field Name="ConversionGroup" Type="ConversionGroup" Null="allow"/>
```

Comment:
It's strange to have a `Conversion` in `BoundConversion` and also inside `BoundConversion.ConversionGroup`. I assume they are the same.

Reply:
Consider adding a comment on the meaning of `null` `ConversionGroup`.

---
In reply to: [189099836](https://github.com/dotnet/roslyn/pull/26356#discussion_r189099836) [](ancestors = 189099836)

Reply:
The `Conversion` in `ConversionGroup` is the object returned from `ClassifyConversionFromExpression`, etc. That `Conversion` instance may be represented as a sequence of `BounConversion` instances, all sharing the same `ConversionGroup` but with distinct individual `Conversion` values.

---
In reply to: [189100739](https://github.com/dotnet/roslyn/pull/26356#discussion_r189100739) [](ancestors = 189100739,189099836)